webapp-django/crashstats/signature/templates/signature/signature_summary.html

```diff
@@ -0,0 +1,171 @@
+<div class="panel">
+    <header class="title">
+        <h2>Operating System</h2>
+    </header>
+    <div class="content">
+        <table class="tablesorter data-table">
+            <thead>
+                <tr>
+                    <th class="header">Rank</th>
+                    <th class="header">Operating System</th>
+                    <th class="header">Count</th>
+                    <th class="header">%</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for platform in query.facets.platform %}
+                <tr>
+                    <td>{{ loop.index }}</td>
+                    <td>{{ platform.term }}</td>
+                    <td>{{ platform.count }}</td>
+                    <td>{{ (platform.count / query.total * 100.0) | round(2) }}</td>
```

Comment:
Unless you have a strong reason to keep it, I would urge that we switch to a single decimal point. The 2 decimal notation is defacto standard for monetary display. E.g. "$123.45". 
If the accuracy is so important then 2 decimal points is far too little, like for scientific measurements. 

I would advocate we switch to 1 decimal point. Especially for percentages.